src/omdata.h

```diff
@@ -22,6 +22,27 @@ struct overmap_spawns {
     int max_population;
     int chance;
 };
+ //terrain flags enum! this is for tracking the indices of each flag.
+    //is_asphalt, is_building, is_subway, is_sewer, is_ants,
```

Comment:
To make them easier to find for people reading through the code. I'll probabl remove the default argument from the set_flag function, but I'll leave that until the next pull request.